react/features/welcome/reducer.js

```diff
@@ -0,0 +1,26 @@
+import { ReducerRegistry } from '../base/redux';
+import { HIDE_SIDE_BAR, SHOW_SIDE_BAR } from './actionTypes';
+
+/**
+ * Reduces the Redux actions of the feature features/recording.
+ */
+ReducerRegistry.register('features/welcome', (state = {
+    sideBarVisible: false
+}, action) => {
+    switch (action.type) {
+    case HIDE_SIDE_BAR:
```

Comment:
I suggest you fold these 2 actions into 1: SET_SIDEBAR_VISIBILITY, same as we do with the toolbox or fimlstrip.